index.css

```diff
@@ -1,3 +1,7 @@
 h1{
     color: brown;
+}
+
+li {
+ list-style: none;
```

Comment:
Nice work eliminating bullet points from the nav links 🎉

Reply:
Thank you 🤝 

Happy Coding!